### ISSUE TITLE
Viewing 'saved chunks', before starting a world download, crashes the game!

### ISSUE THREAD

**Issue by antofthy** [Pokechu22/WorldDownloader#123]
Title says it all..  the game will crash if you try to view the saved chunks map,
before you actually start downloading the world for the first time.

Once you do start downloading, it works fine, even if no actual chunks have been written to the disk as yet.  As soon as you have written some chunks (you moved around, or stopped the world download) then of course it shows those chunks.  Just as it should.

**Comment by Pokechu22** [comment]
Which Minecraft version?  I ran into that while working on 1.14, but I thought I fixed it... can you post the crash report?

**Comment by antofthy** [comment]
1.13.2  Just downloaded it last night, and I use it via MultiMC to add the zip to the base image.

It isn't a big problem.  Sort of nonsensical usage.  But the world is full of dumb people (even me) :-)

**Comment by Pokechu22** [comment]
Looks like I already fixed the _crash_ in 07c3f3db47414238ed73160fa084cb57ec46f3b7 (for the next update, not part of the one released a few days ago), but the GUI itself doesn't display anything until you've started download.  This actually is less than optimal, because the ideal behavior would be for it to show areas you saved before if you've used the world in the past (including ones far away from your current position), but I guess as it is currently you'd have to start downloading first (which is definitely confusing).

The whole saved chunks GUI is currently somewhat incomplete -- I'd been working on it before starting work on 1.14, and didn't finish it up before releasing.

**Comment by Pokechu22** [comment]
The crash is fixed in [version 4.0.5.1](https://github.com/Pokechu22/WorldDownloader/releases/tag/v4.0.5.1).
